examples/unity-simple-examples/safe_policy.py

```diff
@@ -152,27 +113,20 @@ def step(self, obs, state=None, mask=None, deterministic=False):
         drone_info = obs[0]
 
         # if the action is safe, use it. otherwise, check another possible action
-        if is_action_safe(drone_info, action[VEL_X])[0]:
-            self.__plot_new_safe_position(drone_info, action[VEL_X])
-        else:
+        if not is_action_safe(drone_info, action[VEL_X])[0]:
             safe_actions = []
             for i in range(max_tries):
```

Comment:
`i` is not used in the for loop, instead you can use `_`

```suggestion
            for _ in range(max_tries):
```